tests/Homely.Storage.Blobs.Tests/AddAsyncTests.cs

```diff
@@ -140,5 +141,68 @@ public async Task GivenSomeUri_AddAsync_AddsTheUri()
                 memoryStream.Length.ShouldBeGreaterThan(0);
             }
         }
+
+        [Theory]
+        [InlineData(CacheControlType.NoCache)]
+        [InlineData(CacheControlType.NoStore)]
+        public async Task GivenAnObjectThatDoesNotExist_AddAsync_SetWithCache_AddsObjectAndCacheIsSet(CacheControlType expectedCacheControl)
+        {
+            // Arrange.
+            var azureBlob = await GetAzureBlobAsync();
+
+            // Act.
+            var blobId = await azureBlob.AddAsync(TestUser, cacheControlType: expectedCacheControl);
+
+            // Assert.
+            blobId.ShouldNotBeNullOrEmpty();
+            var blob = await azureBlob.GetAsync<SomeFakeUser>(blobId, new List<string> { "CacheControl" }, default);

```

Comment:
make the `new List<string> { "CacheControl" }` a private static variable, at the top of the file. then this (and other calls) can use the same instance.

Reply:
🔧